### PR TITLE
Ensure built-in poster has default background color

### PR DESCRIPTION
**Pull request by cdata**
I think this may have regressed as part of the seamless poster change in #890 . I don't think it's necessary for the poster to have a fallback color of `inherit` for seamless poster transitions to work, and also it means that in most cases, the built-in poster won't have any background color at all.

### CODE REVIEW

**Comment by elalish** [comment]
Hmm, I think the issue is less seamless posters and more transparent rendering (and therefore transparent posters). Look at modelviewer.dev/examples/lazy-loading.html with this change. You'll see that the background color changes from white to grey now, which is why I put `inherit` IIRC.

**Comment by cdata** [comment]
Ah, so the poster color needs to be transparent in the case that there is a configured poster image with transparency in it? I think it's fair to say that this should be configured by the user as needed to achieve the desired effect.

File: packages/modelviewer.dev/examples/lazy-loading.html [hunk 1]
```diff
@@ -81,6 +81,11 @@ <h2 class="demo-title">Display a poster until loaded</h2>
           <h4></h4>
         </div>
         <example-snippet stamp-to="demo-container-1" highlight-as="html">
+          <style>
+            model-viewer#reveal {
+              --poster-color: transparent;
+            }
+          </style>
```
Comment (elalish) [inline packages/modelviewer.dev/examples/lazy-loading.html:88]: I think we should move this inside the template to make it a visible part of our documentation. We should also add this to the example below. The third example seems to already work without needing this, even though it is also a transparent poster. Do you know why?

**Comment by cdata** [comment]
Many docs and an example have been updated

File: packages/modelviewer.dev/examples/lazy-loading.html (hunk 1 above)
Comment (cdata) [reply packages/modelviewer.dev/examples/lazy-loading.html:88]: Oh, sorry. This was a mistake. It should be in the template.

Comment (cdata) [reply packages/modelviewer.dev/examples/lazy-loading.html:88]: The third example replaces the default poster (via a `slot`-ed alternative), so `--poster-color` no longer factors into the equation.

Comment (elalish) [reply packages/modelviewer.dev/examples/lazy-loading.html]: I think we still want to add this change to the second example, as it is like the first.

**Pull request merged**